### PR TITLE
[STORY-678] feat(db): Add new db dashboard documentation [PG PART]

### PR DESCRIPTION
**Pull request by baptmaury**
Wording is based on https://github.com/Scalingo/documentation/pull/2846

I didn't added Maintenance part as it's common to all addons

### CODE REVIEW

**Review by Anouchka-M — CHANGES_REQUESTED** [review]

File: src/_posts/databases/postgresql/2000-01-01-accessing.md [hunk 1]
```diff
@@ -187,8 +187,8 @@ recommend activating Internet Accessibility.
 
 1. Make sure [you have TLS enforced]({% post_url databases/postgresql/2000-01-01-connecting %}#enforcing-tls-connection)
 2. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
-3. Select the **Overview** tab
-4. Locate the **TLS/Internet Access** block
+3. Select the **Settings** tab
+4. In the **Settings** menu, select **Internet Access** (settings/networking)
 5. Enable the **Internet Accessibility** option
 6. The database is now available using the corresponding [connection URI]({% post_url databases/postgresql/2000-01-01-connecting %})
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-accessing.md]:
```suggestion
4. In the Settings submenu, select **Internet Access**
5. Locate the **Internet Accessibility** block
6. Click the **Enable** button
7. The database is now available using the corresponding [connection URI]({% post_url databases/postgresql/2000-01-01-connecting %})
```

File: src/_posts/databases/postgresql/2000-01-01-backing-up.md [hunk 2]
```diff
@@ -72,10 +72,10 @@ the workload on our infrastructure.
...
 1. From your web browser, open your [database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
 2. Click the **Backups** tab
-3. Locate the **Backup Settings** block
-4. Make sure the **Periodic backups** toggle is set to `enabled`
-5. Chose the time and select the timezone
-6. Click the **Set** button to validate
+3. Locate the **Backup schedule** block
+4. Click on the **Schedule** button and make sure to check **I want to enable scheduled backups**
+5. Chose the time in UTC
+6. Click the **Update** button to validate
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-backing-up.md:78]:
```suggestion
4. Click the **Schedule** button
5. Make sure to check the **I want to enable scheduled backups** checkbox
6. Pick an hour (**timezone is UTC**)
7. Validate by clicking the **Update** button
```

File: src/_posts/databases/postgresql/2000-01-01-connecting.md [hunk 3]
```diff
@@ -102,9 +102,10 @@ If you want to enforce TLS connections to your database so that TLS is not only
...
-2. Select the **Overview** tab
-3. Locate the **TLS/Internet Access** block
-4. Toggle the corresponding option:
+2. Select the **Settings** tab
+3. Click on the **Internet Access** menu on the left
+4. Locate the **Force TLS connections** block
+5. Toggle the corresponding option:
 
 {% assign img_url = "https://cdn.scalingo.com/documentation/screenshot_database_mongo_force_tls.png" %}
 {% include mdl_img.html %}
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-connecting.md:111]: I'd remove the screenshot and just put that as the last instruction:
```suggestion
5. Click the **Enable** button
```

File: src/_posts/databases/postgresql/2000-01-01-managing.md [hunk 4]
```diff
@@ -132,12 +132,10 @@ In both cases, once the operation is finished, the application is restarted.
...
-1. From your web browser, open your [database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
-2. Click the **Overview** tab
-3. Locate the **Database Upgrade** block
-4. If an upgrade is available, the text in the block explains what will be
-   done.
-5. To launch the upgrade, click the **Upgrade to ...** button
+1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
+2. Click on the **Settings** tab. Locate the **Database Upgrade** block
+3. In **General**, if an upgrade is available, the text in the block explains what will be
+4. To launch the upgrade, click the **Upgrade to ...** button
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-managing.md]:
```suggestion
2. Select the **Settings** tab
3. In the **Settings** submenu, select the **General** tab
4. Locate the **Database Version** block
5. If an upgrade is available, a button allows you to trigger the upgrade
6. Click the button to launch the upgrade process
```

File: src/_posts/databases/postgresql/2000-01-01-managing.md [hunk 5]
```diff
@@ -180,10 +178,10 @@ ALTER DEFAULT PRIVILEGES FOR USER <database> IN SCHEMA public GRANT ALL ON SEQUE
...
 #### Using the Database Dashboard
 
-1. From your web browser, open your [database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-postgresql-dashboard)
-2. Click the **Users** tab
-3. The list of available users is displayed under the **User Management**
-   section
+1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-postgresql-dashboard)
+2. Click on the **Settings** tab
+3. In the **Settings** menu, select **Users**
+4. The list of available users is displayed under the **User Management**
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-managing.md]:
```suggestion
2. Click the **Settings** tab
3. In the **Settings** submenu, select **Users**
4. The list of available users is displayed in the **User Management** block
```

File: src/_posts/databases/postgresql/2000-01-01-managing.md [hunk 6]
```diff
@@ -223,14 +221,14 @@ method used:
 
 #### Using the Database Dashboard
 
-1. From your web browser, open your [database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
-2. Click the **Users** tab
-3. Fill the **Add a new user** form:
+1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
+2. Click on the **Settings** tab
+3. In the **Settings** menu, select **Users**
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-managing.md]:
```suggestion
2. Click the **Settings** tab
3. In the **Settings** submenu, select **Users**
```

File: src/_posts/databases/postgresql/2000-01-01-managing.md [hunk 7]
```diff
@@ -292,12 +290,13 @@ Protected users password cannot be updated
...
-3. Locate the user you want to remove
-4. Click the **Delete** button next to the user
-5. Confirm the removal
-
+1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
+2. Click on the **Settings** tab
+3. In the **Settings** menu, select **Users**
+4. Locate the user you want to remove
+5. Click the **"..."** button next to the user and select **Delete user**
+6. Confirm the deletion by typing the name of the user in the popup window
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-managing.md]:
```suggestion
2. Click the **Settings** tab
3. In the **Settings** submenu, select **Users**
4. Locate the user you want to remove
5. Click the **…** button next to the user and select **Delete user** from the popup menu
6. In the popup window, confirm the deletion by typing the name of the user
```

File: src/_posts/databases/postgresql/2000-01-01-managing.md [hunk 8]
```diff
@@ -316,6 +315,16 @@ but you need to enable those manually according to your needs.
 
 Please refer to the Overview page to get the [list of available extensions]({% post_url databases/postgresql/2000-01-01-overview %}#available-extensions).
 
+### Show the enabled extensions
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-managing.md:326]:
```suggestion
### Viewing Enabled Extensions
```

File: src/_posts/databases/postgresql/2000-01-01-managing.md [hunk 9]
```diff
@@ -316,6 +315,16 @@ but you need to enable those manually according to your needs.
...
+
+#### Using the Database Dashboard
+
+1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
+2. Click on the **Settings** tab
+3. In the **Settings** menu, select **Condiguration**
+4. Locate the **PostgreSQL Extensions**
+5. The list of enabled extensions will appear in small pills
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-managing.md]:
```suggestion
2. Click the **Settings** tab
3. In the **Settings** submenu, select **Configuration**
4. Locate the **PostgreSQL Extensions** block
5. The list of enabled extensions is displayed here
```

File: src/_posts/databases/postgresql/2000-01-01-restoring.md [hunk 10]
```diff
@@ -20,8 +20,9 @@ database to be completely stopped, causing an inevitable downtime.
 1. From your web browser, open your [database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
 2. Click the **Backups** tab
 3. Locate the **Point-in-Time Recovery** block
-4. Pick a date and time
-5. Click the **Restore Database** button
+3. Click on **Start a PITR** button
+4. Pick a date and time (in UTC) and check **I understand that this action will permanently delete existing data and cannot be cancelled or undone once started.**
+5. Click the **Confirm** button
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-restoring.md:25]:
```suggestion
3. Click the **Start a PiTR** button
4. Pick a date and time (**timezone is UTC**)
5. Make sure to check the **I understand that this action will permanently delete existing data and cannot be cancelled or undone once started.** checkbox
6. Validate by clicking the **Confirm** button
```

File: src/_posts/databases/postgresql/2000-01-01-accessing.md [hunk 11]
```diff
@@ -187,8 +187,8 @@ recommend activating Internet Accessibility.
 
 1. Make sure [you have TLS enforced]({% post_url databases/postgresql/2000-01-01-connecting %}#enforcing-tls-connection)
 2. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
-3. Select the **Overview** tab
-4. Locate the **TLS/Internet Access** block
+3. Select the **Settings** tab
+4. In the **Settings** submenu, select **Internet Access** (settings/networking)
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-accessing.md:191]:
```suggestion
4. In the **Settings** submenu, select **Internet Access**
```

**Review by Frzk — CHANGES_REQUESTED** [review]
I've made a bunch of suggestions.
IMHO, some are required. Some others may be more arguable, I'll let you decide.

Nice work on the dashboard <3

File: src/_posts/databases/mysql/2000-01-01-accessing.md [hunk 12]
```diff
@@ -178,9 +178,10 @@ recommend activating Internet Accessibility.
 1. Make sure [you have TLS enforced]({% post_url databases/mysql/2000-01-01-connecting %}#enforcing-tls-connection)
 2. From your web browser, [open your database dashboard]({% post_url databases/mysql/2000-01-01-getting-started %}#accessing-the-scalingo-for-mysql-dashboard)
 3. Select the **Overview** tab
-4. Locate the **TLS/Internet Access** block
-5. Enable the **Internet Accessibility** option
-6. The database is now available using the corresponding [connection URI]({% post_url databases/mysql/2000-01-01-connecting %})
+4. In the Settings submenu, select **Internet Access**
```
Comment (Frzk) [inline src/_posts/databases/mysql/2000-01-01-accessing.md]:
```suggestion
4. In the **Settings** submenu, select **Internet Access**
```

File: src/_posts/databases/postgresql/2000-01-01-connecting.md [hunk 13]
```diff
@@ -102,12 +102,10 @@ If you want to enforce TLS connections to your database so that TLS is not only
...
 
 1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
-2. Select the **Overview** tab
-3. Locate the **TLS/Internet Access** block
-4. Toggle the corresponding option:
-
-{% assign img_url = "https://cdn.scalingo.com/documentation/screenshot_database_mongo_force_tls.png" %}
-{% include mdl_img.html %}
+2. Select the **Settings** tab
+3. In the Settings submenu, select **Internet Access**
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-connecting.md:106]:
```suggestion
3. In the **Settings** submenu, select **Internet Access**
```

File: src/_posts/databases/postgresql/2000-01-01-managing.md [hunk 14]
```diff
@@ -37,7 +37,7 @@ your data and fits your application workload.
 2. Click on the application for which you want to scale the Scalingo for
    PostgreSQL® addon
 3. Click on the **Resources** tab
-4. Locate the **Addons** block and click on the **...** button
+4. Locate the **Addons** block and click on the **"&#8230;"** button
 5. From the dropdown menu, select **Change plan**
 6. Select the new plan
 7. Click the **Finish** button
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-managing.md:43]:
```suggestion
4. Locate the **Addons** block
5. Click the **"&#8230;"** button
6. From the dropdown menu, select **Change plan**
7. Select the new plan
8. Validate by clicking the **Finish** button
```

File: src/_posts/databases/postgresql/2000-01-01-managing.md [hunk 15]
```diff
@@ -223,14 +224,19 @@ method used:
...
+3. In the **Settings** submenu, select **Users**
+4. Fill the **Add a user** form:
    - Fill a username
-   - Fill the corresponding password twice
-4. (optional) If you want to restrict this user to read only abilities, make
-   sure to check the **Read only** checkbox
-5. Validate by clicking the **Add User** button
+5. (optional) If you want to allow this user to have writings abilities, make
+   sure to check the **Write** checkbox
+6. Validate by clicking the **Add this user** button
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-managing.md:234]:
```suggestion
   - (optional) To allow the new user to have writings abilities, make
     sure to check the **Write (optional)** checkbox
5. Validate by clicking the **Add this user** button
```

File: src/_posts/databases/postgresql/2000-01-01-managing.md [hunk 16]
```diff
@@ -261,7 +267,8 @@ method used:
 ### Updating a User Password
 
 {% note %}
-Updating a User Password is only available from the command line
+Updating a User Password is only available from the command line.
+Protected users password cannot be updated
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-managing.md:271]:
```suggestion
Protected users password cannot be updated.
```

File: src/_posts/databases/postgresql/2000-01-01-managing.md [hunk 17]
```diff
@@ -284,20 +291,21 @@ Updating a User Password is only available from the command line
    User "my_user" created with password "YANs3y07m5_KJC2MSDGebh8tx1lliFWh2Yb239zVqGQvbElWDjIN7QWspVH92Ul8".
    ```
 
+### Deleting a User
+
 {% note %}
-Protected users password cannot be updated
+Protected users cannot be deleted
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-managing.md:297]:
```suggestion
Protected users cannot be deleted.
```

File: src/_posts/databases/postgresql/2000-01-01-managing.md [hunk 18]
```diff
@@ -316,6 +323,16 @@ but you need to enable those manually according to your needs.
...
 
+### Viewing Enabled Extensions
+
+#### Using the Database Dashboard
+
+1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
+2. Click the **Settings** tab
+3. In the **Settings** submenu, select **Configuration**
+4. Locate the **PostgreSQL Extensions** block
+5. The list of enabled extensions is displayed here
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-managing.md:334]:
```suggestion
4.  The list of enabled extensions is displayed in the **PostgreSQL Extensions** block
```

File: src/_posts/databases/postgresql/2000-01-01-managing.md [hunk 19]
```diff
@@ -284,20 +291,21 @@ Updating a User Password is only available from the command line
...
-4. Click the **Delete** button next to the user
-5. Confirm the removal
-
+1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
+2. Click the **Settings** tab
+3. In the **Settings** submenu, select **Users**
+4. Locate the user you want to remove
+5. Click the **…** button next to the user and select **Delete user** from the popup menu
+6. In the popup window, confirm the deletion by typing the name of the user
+7. Validate by clicking the **Confirm** button
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-managing.md:308]:
```suggestion
5. Click the **&#8230;** button next to the user
6. From the popup menu, select **Delete user**
7. In the popup window, confirm the deletion by typing the name of the user
8. Validate by clicking the **Confirm** button
```

**Review by Frzk — CHANGES_REQUESTED** [review]
Almost there :smile_cat:

File: src/_posts/databases/postgresql/2000-01-01-accessing.md (hunk 1 above)
Comment (baptmaury) [reply src/_posts/databases/postgresql/2000-01-01-accessing.md]: already implemented

File: src/_posts/databases/postgresql/2000-01-01-managing.md (hunk 5 above)
Comment (baptmaury) [reply src/_posts/databases/postgresql/2000-01-01-managing.md]: already implemented

File: src/_posts/databases/postgresql/2000-01-01-managing.md (hunk 6 above)
Comment (baptmaury) [reply src/_posts/databases/postgresql/2000-01-01-managing.md]: already implemented

File: src/_posts/databases/postgresql/2000-01-01-managing.md (hunk 7 above)
Comment (baptmaury) [reply src/_posts/databases/postgresql/2000-01-01-managing.md]: already implemented

File: src/_posts/databases/postgresql/2000-01-01-managing.md (hunk 9 above)
Comment (baptmaury) [reply src/_posts/databases/postgresql/2000-01-01-managing.md]: already implemented

File: src/_posts/databases/postgresql/2000-01-01-managing.md [hunk 20]
```diff
@@ -37,10 +37,11 @@ your data and fits your application workload.
...
 3. Click on the **Resources** tab
-4. Locate the **Addons** block and click on the **...** button
-5. From the dropdown menu, select **Change plan**
-6. Select the new plan
-7. Click the **Finish** button
+4. Locate the **Addons** block
+5. Click the **"&#8230;"** button
+6. From the dropdown menu, select **Change plan**
+7. Select the new plan
+8. Validate by clicking the **Finish** button
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-managing.md:44]: Sorry this one is my fault :-(
```suggestion
8. Click the **Finish** button
```

File: src/_posts/databases/postgresql/2000-01-01-managing.md [hunk 21]
```diff
@@ -223,14 +225,19 @@ method used:
...
-   sure to check the **Read only** checkbox
-5. Validate by clicking the **Add User** button
+   - (optional) To allow the new user to have writings abilities, make
+     sure to check the **Write (optional)** checkbox
+5. Validate by clicking the **Add this user** button
+
+{% note %}
+A secure password will be generated automatically. You will be able to view and copy it only once.
+It will appear in a popup window after the user creation.
+{% endnote %}
```
Comment (Frzk) [inline src/_posts/databases/postgresql/2000-01-01-managing.md:240]:
```suggestion
{% note %}
A secure password is generated automatically and shown in a popup window after the user creation.
You will be able to view and copy it only once.
{% endnote %}
```

**Review by Frzk — CHANGES_REQUESTED** [review]

File: src/_posts/databases/postgresql/2000-01-01-managing.md (hunk 21 above)
Comment (baptmaury) [reply src/_posts/databases/postgresql/2000-01-01-managing.md]: Implemented

**Review by Frzk — APPROVED** [review]
LGTM :tada: 
Well done :)

**Review by Anouchka-M — APPROVED** [review]

**Pull request merged**